pkg/machines/scripts/create_machine.sh

```diff
@@ -19,12 +20,16 @@ handleFailure(){
 }
 
 # prepare virt-install parameters
-COMPARISON=$(awk 'BEGIN{ print "'$STORAGE_SIZE'"<=0 }')
-if [ "$COMPARISON" -eq 1 ]; then
-    # default to no disk if size 0
-    DISK_OPTIONS="none"
+if [ "$SOURCE_TYPE" == "disk_image" ]; then
+    DISK_OPTIONS="$SOURCE,device=disk"
```

Comment:
This reminds me, we don't set `virtio` anywhere, nor do we have a way to specify that. It should hopefully not matter for most disk images, as a proper OS uses UUIDs or labels in fstab instead of device names (sda vs. vda). So, nothing for this PR, but should we at some point consider defaulting to virtio for better performance?

Reply:
When the operating system is selected and not left undefined in the VM creation dialog the virtio disk type is chosen by virt-install.
Other than that, I am not sure that virtio disks are supported by every OS, so hardcoding it here should not be correct. For example Windows does not come with virtio drivers.

Reply:
Right, I didn't mean to suggest hardcoding it here. Great to hear that virtio gets chosen as a default dynamically!